homework-submission/todo-app/src/components/todoStatic.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+const TodoStatic = () => {
```

Comment:
In the homework you are asked to use for the static list the same components as for the dynamic list. So to render the list item using your `<Todo>` component.

Reply:
@ArcoMul Thanks a lot to review my homework and give some feedback! Then I'll delete the todoStatic component and I'll put all the list (static & dynamic) in one component!